libdebug/debugger/internal_debugger.py

```diff
@@ -279,6 +291,9 @@ def detach(self: InternalDebugger) -> None:
 
         self._join_and_check_status()
 
+        if self.fast_memory:
```

Comment:
if the memory access methods are interchangeable, this is not a reliable check

Reply:
[bc9db46](https://github.com/libdebug/libdebug/pull/92/commits/bc9db4634e70d9a692ded656cadc8fa8ed675d20)